metabase/metabase
# Add Icon for entity search

Closes https://github.com/metabase/metabase/issues/30482

### Description

Adds a new index icon for entity index items. 

![Screen Shot 2023-05-02 at 11 56 33 AM](https://user-images.githubusercontent.com/30528226/235746980-6b259c23-b556-4e81-b43c-efbd2903c945.png)

### How to verify

Add an entity index. Search for an item in the index and see the icon in search




- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30491)
